### ISSUE TITLE
Add unit tests for the rest of the primitives classes

### ISSUE THREAD

**Issue by MathewMacDougall** [UBC-Thunderbots/Software#135]
**Is your feature request or enhancement related to a problem? What specifically could be better? Please describe.**
All `Primitive` classes should have unit tests.

**Describe the solution you'd like**
Add unit tests for all the following primitives. 

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered. Since test for the `MovePrimitive` (#129 ) are already done, we can use that as a template/example. See #131 
* Catch primitive. Blocked by #125 
* Dribble primitive. Blocked by #104 
* DirectWheels primitive. Blocked by #92 
* DirectVelocity primitive. Blocked by #91 
* Stop primitive. Blocked by #90 
* MoveSpin primitive. Blocked by #89 
* Pivot primitive. Blocked by #88 
* Kick and chip primitives. Blocked by #87
